guides/asynchronous-tasks/readme.md

```diff
@@ -359,44 +382,48 @@ The purpose of transient tasks is when a task is an implementation detail of an
 - A background worker or batch processing job which is independent of any specific operation, and is lazily created.
 - A cache system which needs periodic expiration / revalidation of data/values.
 
-Bearing in mind, in all of the above cases, you may need to validate that the background task hasn't been stopped, e.g.
+Bear in mind, in all of the above cases, you may need to validate that the background task hasn't been stopped, e.g. ???
```

Comment:
@ioquatix Can you suggest how to finish this sentence where the ??? is? I wasn't sure.

Reply:
I think we can delete the sentence, IIRC, what I was trying to say, was that such tasks could be stopped due to going out of scope, rather than explicitly being `Task#stop`ed. In other words, `@refresh ||= Async(transient: true) {...}` is required to restart tasks if they go out of scope and have been stopped - "you may need to validate the background task hasn't been stopped".

A more specific example: let's say you had an in-process background job processing system. When you schedule a job, you should ensure it's running when adding the job. Otherwise, it may stop as a matter of program flow, and equally, it must be started as a matter of program flow.

This is captured in the below example.

Reply:
Sentence deleted.